packages/cacti-copm-core/src/main/proto/generated/openapi/models/view_v1_pb.proto

```diff
@@ -0,0 +1,25 @@
+/*
+  Hyperledger Cacti Plugin - Common Operator Primitives
+
+  Contains/describes the Hyperledger Cacti Common Operator Primitives plugin.
+
+  The version of the OpenAPI document: 2.0.0-rc.2
+
+  Generated by OpenAPI Generator: https://openapi-generator.tech
+*/
+
+syntax = "proto3";
+
+package org.hyperledger.cacti.plugin.cacti.plugin.copm.core;
+
+import "google/protobuf/any.proto";
+
+import "models/view_address_v1_pb.proto";
+
+message ViewV1PB {
```

Comment:
View contains response to the remote query right?

Reply:
No, this is the inbound message.